src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IepSummaryDateTest.kt

```diff
@@ -244,7 +245,24 @@ class IepSummaryDateTest {
       )
 
       assertThat(iepSummary.daysSinceReview).isEqualTo(30)
-      assertThat(iepSummary.daysOnLevel()).isEqualTo(90)
+      assertThat(daysOnLevel(iepSummary.iepDetails)).isEqualTo(90)
+    }
+
+    @Test
+    fun `days on level cannot be calculated when iepDetail history is missing`() {
+      val iepTime = LocalDateTime.now().minusDays(3)
+      val iepSummary = IepSummary(
+        bookingId = 1L,
+        iepDate = iepTime.toLocalDate(),
+        iepLevel = "Basic",
+        iepTime = iepTime,
+        iepDetails = emptyList(),
+      )
+
+      assertThat(iepSummary.daysSinceReview).isEqualTo(3)
+      assertThatThrownBy { daysOnLevel(iepSummary.iepDetails) }
```

Comment:
the `iepSummary` object is not needed but it demonstrates how `daysOnLevel` is actually used in main code